spec/index.adoc

```diff
@@ -4109,14 +4126,15 @@ callbacks = λ(callbacks, response, refunded_cycles, sysenv, available) → λ w
     if typeof(func) ≠ func (i32) -> () then Trap
 
     let es = ref { empty_execution_state with
```

Comment:
Is this execution state fully defined? E.g., there are no params updated.

Reply:
At least that seems to be consistent with this part of the textual spec:
> During the execution of the cleanup function, only a subset of the System API is available (namely ic0.debug_print, ic0.trap and the ic0.stable_* functions). The cleanup function is expected to run swiftly (within a fixed, yet to be specified cycle limit) and serves to free resources associated with the callback.

The canister doesn't seem to have access to that data in the cleanup callback anyway.

Reply:
So is this resolved now? @bogwar